services/graphite/parser.go

```diff
@@ -50,7 +50,9 @@ func NewParserWithOptions(options Options) (*Parser, error) {
 		filter := ""
 		// Format is [filter] <template> [tag1=value1,tag2=value2]
 		parts := strings.Fields(pattern)
-		if len(parts) >= 2 {
+		if len(parts) < 1 {
+			continue
```

Comment:
yep, it panic'ed if the user supplied an empty string template, which I found out completely on accident